ios/RNSScreenStack.mm

```diff
@@ -102,16 +154,21 @@ - (instancetype)initWithManager:(RNSScreenStackManager *)manager
   return self;
 }
 
-- (UIViewController *)reactViewController
```

Comment:
why is it now `paper specific`?

Reply:
It does not have to be I guess. It was just never used (and not present) in Fabric impl. I will move it to common section.